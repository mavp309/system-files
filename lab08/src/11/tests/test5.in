Karan
24
77889900
82
