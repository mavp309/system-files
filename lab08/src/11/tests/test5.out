Name: Karan
Age: 24
Roll: 77989900
Marks: 82
